src/xtgeo/grid3d/_grid_roxapi.py

```diff
@@ -299,8 +323,13 @@ def _convert_to_xtgeo_grid_v2(roxgrid, gname):
 
 
 def export_grid_roxapi(
-    self, projectname, gname, realisation, info=False, method="cpg"
-):  # pragma: no cover
+    self: Grid,
+    projectname: str,
+    gname: str,
+    realisation: int,
+    info: bool = False,
+    method: str = "cpg",
```

Comment:
What to do here? I was considering using `Literal["cpg"] | Any` as the code in current state will only do something specific if method=="cpg".

Reply:
I think `str | Literal["cpg"]` would be good